### PR TITLE
fix: [Auto Routing Improved] feature testing may not find controller/method

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://forum.codeigniter.com/showthread.php?tid=87818

- fix `AutoRouterInterface::getRoute()`

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
All checks passed!

**Comment by kenjis** [comment]
Can someone please review this?

**Review by michalsn — APPROVED** [review]

File: system/Router/RouteCollection.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ class RouteCollection implements RouteCollectionInterface
     /**
      * The current method that the script is being called by.
      *
-     * @var string
+     * @var string HTTP verb (lower case) or `*`
```
Comment (samsonasik) [inline system/Router/RouteCollection.php:153]: can this include possible verb value? eg: `get|post`...

Comment (kenjis) [reply system/Router/RouteCollection.php]: Added.

**Review by samsonasik — APPROVED** [review]

**Pull request merged**